src/main/terminal.js

```diff
@@ -1,11 +1,11 @@
 'use babel';
 var vscode = require('vscode');
-import Logger from '../helpers/logger.js'
-import ApiWrapper from '../main/api-wrapper.js';
+const Logger = require('../helpers/logger.js')const Logger = require('../helpers/logger.js')
```

Comment:
I think this is an issue?

Reply:
Yes. That look like trouble. I'll get it fixed.